exonum-java-binding/exonum_launcher_java_plugins/exonum_instance_configuration_plugin/plugin.py

```diff
@@ -76,6 +92,39 @@ def serialize_config_impl(self, config: Any, instance_name: str) -> bytes:
 
             return result
 
+    def serialize_standard_config(self, config: Any, instance_name: str) -> bytes:
+        configuration_message = service_pb2.ServiceConfiguration()
+
+        configuration_message.value = self.extract_value(config, instance_name)
+        configuration_message.format = self.extract_format(config, instance_name)
+
+        return configuration_message.SerializeToString()
+
+    @staticmethod
+    def extract_value(config: Any, instance_name: str) -> str:
+        if InstanceSpecLoader.config_field_exists(config, _VALUE_FIELD_NAME):
+            return config[_VALUE_FIELD_NAME]
+        elif InstanceSpecLoader.config_field_exists(config, _FROM_FILE_FIELD_NAME):
+            file_path = config[_FROM_FILE_FIELD_NAME]
+            with open(file_path, 'r') as file:
+                return file.read()
+        else:
+            InstanceSpecLoader.raise_exception_field_not_found(instance_name, _VALUE_FIELD_NAME)
```

Comment:
Shan't we say in the exception that either value or filename are expected? IIUC, it will only say of value.

Reply:
Fixed